README.md

```diff
@@ -12,21 +12,29 @@
 
 Clone the Git repository
 Install Dependencies
-Run Node
+Run npm
 
 #### `.env` requirements
 
-- `PORT` - 3000
+- MONGO_ATLAS_URI= mongodb+srv://pauld:Djsfpe616@cluster0-ydvvi.mongodb.net/notesy?retryWrites=true&w=majority
 
 #### Running the app
 
-- `node -a <INSERT NOTE TEXT HERE>`
-- `node --add <INSERT NOTE TEXT HERE>`
+- open your terminal and run 
+```npm index.js <action> <message> <category>```
+  Actions - --add(-a), --find(-f), --delete(-d)
+  Message - A message you would like to save
+  Category - A category to 
```

Comment:
This is all coming out in a giant chunk of writing, if you want each of these to be different line then make sure to add an extra space before where you want the line break